cmd/serve.go

```diff
@@ -373,13 +375,20 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 			}
 			serveOpts.SMSMessengerClient = smsMessengerClient
 
+			// Setup default AP Auth enforcer
+			apAPIService, err := di.NewAnchorPlatformAPIService(serveOpts.AnchorPlatformBasePlatformURL, serveOpts.AnchorPlatformOutgoingJWTSecret)
+			if err != nil {
+				log.Ctx(ctx).Fatalf("error creating SMS client: %s", err.Error())
```

Comment:
I believe this error should be something like `error creating Anchor Platform API service`.

Reply:
Addressed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/10/commits/0a380e30939dd57210931c2405f26a8d09ff5986.